doc/devel/gitwash/matplotlib_for_dev.rst

```diff
@@ -0,0 +1,20 @@
+.. _matplotlib-for-dev:
+
+=================================================
+Installing matplotlib from source for development
+=================================================
+
+After obtaining a local copy of the matpotlib source code (:ref:`set-up-fork`),
+navigate to the matplotlib directory and run the following in the shell:
+
+::
+    
+    python setup.py develop
```

Comment:
A vast majority of the work that needs to be done on mpl is in the pure-python layers, so a source install can be very convenient.